pkg/execution/state/redis_state/lua/queue/enqueue.lua

```diff
@@ -42,12 +40,9 @@ local accountID           		= ARGV[9]
 -- Key queues v2
 local enqueueToBacklog				= tonumber(ARGV[10])
 local shadowPartitionItem     = ARGV[11]
-local backlogItemA            = ARGV[12]
-local backlogItemB            = ARGV[13]
-local backlogItemC            = ARGV[14]
-local backlogIdA              = ARGV[15]
-local backlogIdB              = ARGV[16]
-local backlogIdC              = ARGV[17]
+local backlogItem             = ARGV[12]
+local backlogID               = ARGV[13]
+local normalize               = tonumber(ARGV[14]) -- signals if this is part of a normalization
```

Comment:
added a `normalize` flag here so if it's an enqueue for normalization, it skips the idempotency and hsetnx checks, otherwise it'll return early and don't get enqueue properly to the appropriate backlog.